main.go

```diff
@@ -142,9 +141,7 @@ func realMain() {
 	// and wait for the process. Indeed, after the process has been
```

Comment:
Just to check - is the 'wait' in this comment still accurate as the `wg` has moved down the chain?